api/maketdata.go

```diff
@@ -127,10 +131,13 @@ func getTickersHandler(storage storage.Market) func(c *gin.Context) {
 // @Param time_start query int false "Start timestamp" default(1574483028)
 // @Param currency query string false "The currency to show charts" default(USD)
 // @Success 200 {object} blockatlas.ChartData
-// @Router /v1/market/charts [get]
-func makeChartsRoute(router gin.IRouter) {
```

Comment:
is this intended change?

Reply:
no, the old routes were wrong in the swagger file and I made a confusion, fixed now